src/graphforge/datasets/loaders/compression.py

```
@@ -37,8 +82,17 @@ def safe_extract_tar(tar: tarfile.TarFile, extract_to: Path) -> None:
     extract_to_resolved = extract_to.resolve()
 
     for member in tar.getmembers():
-        # Compute target path for this member
-        target_path = extract_to / member.name
+        # Debug: Print member name for Windows debugging
+        import sys
```

Ruff (0.14.14)

[error] Redefinition of unused `sys` from line 15: `sys` redefined here

Remove definition: `sys`

(F811)